server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -595,4 +596,23 @@ public void deletedMessageVaultRestoreShouldCreateATask() throws Exception {
         .then()
             .body("status", is("completed"));
     }
+
+    @Test
+    public void deletedMessageVaultExportShouldCreateATask() throws Exception {
```

Comment:
does it test the task serialization?

Reply:
yes, but anyway I will improve the tests to be more coherent